src/Sulu/Bundle/AdminBundle/SuluAdminBundle.php

```diff
@@ -24,5 +25,14 @@ public function build(ContainerBuilder $container)
 
         $container->addCompilerPass(new AddAdminPass());
         $container->addCompilerPass(new AddMetadataProviderPass());
+
+        if ($container->hasExtension('fos_js_routing')) {
+            if ($this->getContainerExtension() === null) {
+                // Required for phpstan, to make sure that we do not have a null value.
+                throw new \ErrorException('This bundle requires a container extension.');
+            }
+
+            $container->addCompilerPass(new ExposeResourceRoutesPass($this->getContainerExtension()->getAlias()));
```

Comment:
There is no need to give the alias `$this->getContainerExtension()->getAlias()` into it. It just make the compilerpass code more complex. There is no support to overwrite the alias and so we can avoid this whole check for extension and alias and make the code more readable and easier to maintain.

```suggestion
            $container->addCompilerPass(new ExposeResourceRoutesPass();
```

Reply:
The reason I have used the alias for it was, that it is also used in your extension class (see [here](https://github.com/sulu/sulu/blob/916d539fe099126d544f121b19d1fa04faafda24/src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php#L146)). But I am fine with changing it to a static string.

Reply:
I would not do that for new code. We should tackle that in `3.0` to remove also from the Extension class the `getAlias()` call.